README.md

```diff
@@ -10,8 +10,64 @@ The website uses the LAA Legal Adviser API. The code is hosted at https://github
 
 The REST API is https://prod.laalaa.dsd.io/.
 
+## Dependencies
+- [docker](https://www.docker.com/)
+
+## Installation for development
+
+### Tips for developing with docker containers
+
+Code editing - You can edit the code on your local disk, with a local editor, as normal. (You don't have to edit the 
+files inside the Docker container, because your local directory is mounted into container.) When you save a file, it 
+becomes present in the container immediately, and the server restarts.
+
+Browsing the app - Point your local browser at http://localhost:8013/
+
+Log output - watch the output generated by the running app using: `docker logs fala -f` or `docker attach fala`
+
+Please ensure your internet network such as your VPN does not interfere with your build.
+
+To run container:
+
+`./run_local.sh`
+
+This shell script contains commands that build and runs the fala app.
+
+If you're not using `Docker Desktop`, you may want to have two terminals open for the following:
```

Comment:
Oh, I wasn't aware Docker Desktop was an option.

Rather than being unopinionated, I think it's good if Readme recommends a particular way, that we we know works well, can improve together, and swap tips on. So I'd encourage you to add a recommendation on one of the two ways

Reply:
I think this is such a minor preference, I'm going to withdraw it